### ISSUE TITLE
Shouldn't FilmFacade constructor be package private?

### ISSUE THREAD

**Issue by mkrzywanski** [jakubnabrdalik/hentai#7]
Just a simple question. Shouldn't [FilmFacade](https://github.com/jakubnabrdalik/hentai/blob/9903c949e26fc4c665d2147795d4ccf6bfd3d27b/hentai/src/main/java/eu/solidcraft/film/domain/FilmFacade.java#L17) constructor be defined as package private? Right now it is defined as public but client code cannot instantiate it with it because it's dependencies are package private so client code (for example in different package) cannot use it. It can be instantiated only by using Spring and having `FilmConfiguration` class scanned.

**Comment by jakubnabrdalik** [comment]
You are correct, that should be package scoped. Perhaps I should update this sample project.

**Comment by mkrzywanski** [comment]
I can file a PR for this simple change if you want.

**Comment by jakubnabrdalik** [comment]
If you'd like to, sure, but I'm thinking about updating the whole thing to a more up-to-date version (I implement these requirements live a few time every year).
